### PR TITLE
removed duplicated text about SIP archetype

### PR DESCRIPTION
**Pull request by Dzuci**
# Description

Removed documentation duplicates

Fixes # (issue)

## Type of change

Please delete options that are not relevant.
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- Check if documentation is updated properly

**Test Configuration**:
* SIP Framework version(s):
* Other configuration:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing (regression) unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: changelogs/documentation/remove_duplicated_documentation.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "author": "dzuci",
+  "pullrequestId": "440",
```
Comment (vladiIkor) [inline changelogs/documentation/remove_duplicated_documentation.json]: 440?  :)

File: docs/installation.md [hunk 2]
```diff
@@ -28,75 +28,9 @@ Before getting started to develop a first adapter with SIP the following require
 
 ## Create the project
 
-### How to create a SIP Adapter from Maven Archetype
+### How to create a SIP Adapter using SIP Archetype
 
-In order to make setting up a new integration adapter project an easy task, we have created a dedicated SIP (Adapter) Maven archetype, that gives the ability to quickly bootstrap a new integration adapter, along with all needed Maven dependencies and goodies, as well as the recommended, preferred organization of the project.
+Guide for creating a new SIP Adapter from archetype can be found [here](https://ikor-gmbh.github.io/sip-framework/archetype/).
```
Comment (vladiIkor) [inline docs/installation.md:31]: This section now has strange structure. We have one title and subtitle and the content is only link to another page with description.
Perhaps we can make link from ### How to create a SIP Adapter using SIP Archetype, without any additional text. 
Maybe we change it a bit to: ### Creating SIP Adapter using SIP Archetype or something similar.

**Review by vladiIkor — CHANGES_REQUESTED** [review]

**Comment by vladiIkor** [comment]
Please fill in the PR form

File: docs/installation.md (hunk 2 above)
Comment (Nemikor) [reply docs/installation.md:31]: I'd say using the title as a link wouldn't be very user friendly, due to titles already having color similar to anchors, while in regular paragraphs they stand out.

Comment (Dzuci) [reply docs/installation.md:31]: I would prefer to keep it like this.

**Review by vladiIkor — APPROVED** [review]

**Review by Nemikor — APPROVED** [review]

**Pull request merged**